### DISCUSSION TITLE
Screenshots

### DISCUSSION THREAD

**Discussion by Chitusnair** [thawkes70/Tafe22S219#6] [General]
Hi Tony,

Hope you have all the screenshots until preparing the development branch section.

Is it possible for you to share this?

Thanks,
Chitra.

**Comment by thawkes70** [comment]
Hi Chitra, I will attach what I have completed so far, please let me know if you spot any errors. 
[ICTICT449 - ASI - Assignment (Universal Calculator)_Assesment3.docx](https://github.com/thawkes70/Tafe22S219/files/9466743/ICTICT449.-.ASI.-.Assignment.Universal.Calculator._Assesment3.docx)
